src/compiler/checker.ts

```diff
@@ -16381,6 +16385,10 @@ namespace ts {
             else {
                 checkExpression(node.closingElement.tagName);
             }
+        }
+
+        function checkJsxElement(node: JsxElement, _checkMode: CheckMode | undefined): Type {

```

Comment:
Save to remove `_checkMode` now?

Reply:
Yes, but I'll need to add it back in a week or two when I add more specific return types for jsx elements and once again track down all the places I need to flow it through again. I figured I'd just leave it here for later.

Reply:
Might deserve a comment in case Andy sees it and removes it 😉